### PR TITLE
Verify update

### PR DESCRIPTION
**Pull request by aahenry51**

### CODE REVIEW

File: mobile/src/screens/verify/Verify.view.tsx [hunk 1]
```diff
@@ -12,7 +12,16 @@ export const VerifyScreenView: FC<TVerifyScreenViewProps> = ({
...
+      isScrollViewDisabled={true}
+      header={
+        <Header
+          leftIconName="ChevronLeft"
+          iconWidth={size.rg}
+          iconHeight={size.rg}
+          containerStyle={styles.header}
+        />
+      }>
       <Header
```
Comment (mshago) [inline mobile/src/screens/verify/Verify.view.tsx:25]: you don't need the second header right?

**Review by Josetf2396 — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]
I approve this

**Review by mshago — APPROVED** [review]

File: mobile/src/screens/foodDetails/FoodDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React, {FC} from 'react';
+import {EAppNavigationRoutes} from '../../navigation/appNavigation/AppNavigation.type';
+import {TFoodDetailsNavigation} from './FoodDetails.type';
+import {FoodDetailsScreenView} from './FoodDetails.view';
+
+export const FoodDetailsScreen: FC<TFoodDetailsNavigation> = ({navigation}) => {
```
Comment (lasharela) [inline mobile/src/screens/foodDetails/FoodDetails.tsx:6]: navigation is not used

File: mobile/src/screens/foodDetails/FoodDetails.view.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import {styles} from './FoodDetails.style';
+
+export const FoodDetailsScreenView: FC<TFoodDetailsScreenViewProps> = ({
+  title,
+  description = 'food description',
+}) => {
+  const renderAddOn = () => {};
+
+  return (
+    <View>
```
Comment (lasharela) [inline mobile/src/screens/foodDetails/FoodDetails.view.tsx:15]: maybe you'll try Container and head prop instead?

**Review by lasharela — CHANGES_REQUESTED** [review]

**Pull request closed without merging**